dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -305,6 +306,11 @@ def run_validation(self, job):
             error_list.write_all_row_errors(job_id)
             # Update error info for submission
             populate_submission_error_info(submission_id)
+
+            # Update last validated date
+            job.submission.last_validated = datetime.utcnow()
+            sess.commit()
```

Comment:
yup, we need 2 here because `populate_submission_error_info` calls `sum_number_of_errors_for_job_list` which does a query on the Job table